### ISSUE TITLE
Doesn't seem to work with the latest Mystic

### ISSUE THREAD

**Issue by k5jat** [xqtr/JAMTools#1]
I think g00r00 made some changes and these are having issues reading the new dat or something...

**Comment by xqtr** [comment]
Yes. After A39 version of Mystic, some record types in Mystic files have changed. So JAMTools cannot read those files anymore. I will make the updates and release a new version.

**Comment by SpitfireTN** [comment]
Mystic stepped away from the standard JAM messaging for some reason.  I have been battling trying to parse the messages also.  You can see what I was able to display so far at rofbbs.com
